examples/dataset-virtuoso/virtuoso/virtuoso.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
-
+set -e
 cd /data
-until curl --head localhost:15000 ; do echo "Waiting for Sidecar" ; sleep 3 ; done ; echo "Sidecar available";
+if [ "$ISTIO_USE" == "YES" ]; then
```

Comment:
"ISTIO_USE" should be boolean (true or false) instead of "YES" or "NO" literals.
Also variable name "USE_ISTIO" instead of "ISTIO_USE" seems more appropriate in this case.